manifests/t/TexasInstruments/TI-SmartView/TI-CollegePlus/1.3.0.61/TexasInstruments.TI-SmartView.TI-CollegePlus.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 0.4.3.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TexasInstruments.TI-SmartView.TI-CollegePlus
+PackageVersion: 1.3.0.61
+PackageLocale: en-US
+Publisher: Texas Instruments Inc.
+PublisherUrl: https://education.ti.com
+PublisherSupportUrl: https://education.ti.com/en/customer-support
+PrivacyUrl: https://www.ti.com/legal/terms-conditions/privacy-policy.html
+Author: Texas Instruments Inc.
+PackageName: 'TI-SmartView Emulator Software for the TI Collège Plus '
```

```suggestion
PackageName: TI-SmartView™ TI-Collège™ Plus
```
This must match Control Panel.